vic/drivers/shared_image/include/vic_driver_shared_image.h

```diff
@@ -196,7 +196,7 @@ void set_force_type(char *cmdstr, int file_num, int *field);
 void sprint_location(char *str, location_struct *loc);
 void vic_alloc(void);
 void vic_finalize(void);
-void vic_image_run(void);
+void vic_image_run(dmy_struct *dmy_current);
```

Comment:
Confusing that you have an `extern dmy_struct dmy_current` in some routines (e.g. `vic_store()`) and then here it is a pointer.